packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -34,7 +34,30 @@ const Duration _kScrollbarTimeToFade = Duration(milliseconds: 600);
 /// [showTrackOnHover]. The thickness of the track and scrollbar thumb will
 /// become larger when hovering, unless overridden by [hoverThickness].
 ///
-// TODO(Piinks): Add code sample
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// This sample shows [Scrollbar] for a fraction of seconds while scrolling.
+/// If you want to persist [Scrollbar] then set [isAlwaysShown] is true along with
+/// [ScrollController].
+///
+/// ```dart
+///
+/// Widget build(BuildContext context) {
+///   return Scrollbar(
+///     child: GridView.builder(
+///       gridDelegate:
+///           SliverGridDelegateWithFixedCrossAxisCount(crossAxisCount: 3),
+///       itemBuilder: (_, __) {
```

Comment:
```suggestion
///       itemBuilder: (BuildContext context, int index) {
```
The indentation appears a bit off here too.
Also, can we have the grid populated with something a bit more interesting than just some 'data' text?

Reply:
should I use this ``` Text('item $index'),``` ?